src/devtools/client/debugger/src/selectors/visibleColumnBreakpoints.js

```diff
@@ -108,7 +109,12 @@ export function getColumnBreakpoints(positions, breakpoints, viewport, selectedS
   // - the position is in the current viewport
   // - there is atleast one other breakpoint on that line
   // - there is a breakpoint on that line
-  const breakpointMap = groupBreakpoints(breakpoints, selectedSource);
+  const allBreakpoints = [...breakpoints, ...requestedBreakpoints];
+  let positions = uniqBy(
+    allBreakpoints.map(bp => bp.location),
+    ({ sourceId, line }) => `${sourceId}:${line}`
+  );
```

Comment:
should we update the comment above to describe what is happening here?

Reply:
What about:
```
// collect all breakpoint positions but make sure that we don't have 2 positions for the same line in the same source
```

Reply:
works for me